README.md

```diff
@@ -1,4 +1,15 @@
 appengine-maven-archetypes-java
 ===============================
 
-App Engine Maven Archetypes
+* appengine-standard-archetype (GAE Standard, uses Cloud SDK based maven / gradle plugins)
```

Comment:
Yes - they are on the /maven page for GAE standard, we'll be updating the page after deployment happens.